pkg/server/server.go

```diff
@@ -0,0 +1,20 @@
+package server
+
+import (
+	"flag"
+	"time"
+
+	serverww "github.com/weaveworks/common/server"
```

Comment:
Afaik, the import alias is not necessary here, even though the local package is named `server` as well. Functions/variables in this package are referenced without package name, though.